menoh/attribute_completion_and_shape_inference.hpp

```diff
@@ -667,6 +667,22 @@ add_variable_to_table(output(0), dtype_of(input(0)),
 else
 
 
+if(node.op_type == "Identity") {
+    
+    
+    
+    {
+        
+        
+assert(node.input_name_list.size() > 0);
+assert(node.output_name_list.size() > 0);
```

Comment:
Isn't it better to check as follows because `make_identity` assumes both inputs and outputs are one elements.
```
assert(node.input_name_list.size() == 1);
assert(node.output_name_list.size() == 1);
```

Reply:
I see. I'll fix